src/planscape/attributes/models.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.gis.db import models
+from planscape.settings import CRS_FOR_RASTERS
+
+class BaseAttribute(models.Model):
+    """
+    A BaseAttribute has a name and an optional display/region name.
+    """
+    # The name of the attribute.
+    attribute_name: models.CharField = models.CharField(max_length=120)
+
+    # The name of the condition for use in interfaces.
+    display_name: models.CharField = models.CharField(
+        max_length=120, null=True)
+
+    # The region associated with the attribute, drawn from the RegionName enum 
+    # in base/region_name.py.
+    region_name: models.CharField = models.CharField(max_length=120, null=True)
+
+
+class Attribute(models.Model):
```

Comment:
will there be more than 1 Attribute per BaseAttribute? If not, we can probably fold the raster_name field into the parent table

Reply:
For conditions, a single condition could have different types (e.g. raw, normalized, AP).

For attributes, depending on how 30m rasters are served, I can see us having a resolution option in the near future (30m vs 300m).

Not sure whether this justifies a separate Attribute model, though - wdyt?

Reply:
Actually: to keep models simple, I'll remove it. It wouldn't be difficult for someone to add it in the future if necessary.